packages/tizen_rpc_port/example/client/lib/message_client.dart

```diff
@@ -102,9 +102,8 @@ class Message extends ProxyBase {
     final Parcel parcel = Parcel.fromPort(port);
     final int cmd = parcel.readInt32();
     if (cmd != _MethodId.result.id) {
-      print('Received parcel is invalid. $cmd');
+      throw Exception('The received parcel is invalid ($cmd).');
     }
-
     return parcel;
```

Comment:
@upple A code change has been made to auto-generated file. Please review and update the TIDL compiler.